definitions/ext-db2/definition.yml

```diff
@@ -0,0 +1,17 @@
+domain: EXT
```

Comment:
A small comment with a description about the type might be useful for the future

Reply:
Any suggestion? Everything that I can come up with seems not really useful (like: IMB DB2 databases instances)

Reply:
I was thinking of the notice that you added to the PR description actually, about these entities being instances/